docs/user-guide/cost-functions/costautoregressive.md

```diff
@@ -0,0 +1,73 @@
+# Autoregressive model change (CostAutoregressive)
+
+## Description
+
+Let $0<t_1<t_2<\dots<n$ be unknown change points indexes.
+Consider the following piecewise autoregressive model
+
+$$
+    y_t = z_t' \delta_j + \varepsilon_t, \quad \forall t=t_j,\dots,t_{j+1}-1
+$$
+
+where $j>1$ is the segment number, $z_t=[y_{t-1}, y_{t-2},\dots,y_{t-p}]$ is the lag vector,and $p>0$ is the order of the process.
+
+The least-squares estimates of the break dates is obtained by minimizing the sum of squared
+residuals :cite:`ar-Bai2000`. <!--- How to handle citation ?  -->
```

Comment:
@deepcharles still have to find out how to handle this. Maybe it is already done in what you did in the other user guide entries. Will have a look !

Reply:
I made a simple Markdown link, formatted like a citation. See https://github.com/deepcharles/ruptures/blob/mkdocs_code_reference/docs/user-guide/cost-functions/costnormal.md for an example.